src/spas/libs/member-management/feature/src/lib/member-management/mock-users.ts

```diff
@@ -0,0 +1,38 @@
+import { User } from "./user-profile";
+
+export const USERS: User[] = [
+  {
+    fullName: 'Alex Aveldanez',
+    emailAddress: 'alexsaveldanez@gmail.com',
+    personalPhoneNumber: 7708917842,
+    homeAddress: '777 Fake Ave San Diego, CA 92139',
+    emergencyContactName: 'Stephanie',
+    emergencyContactPhone: 7068448739,
+    signUpDate: '1-5-2020',
+    congregationName: 'Southport',
+    groupNumber: 1
+  },
+  {
+    fullName: 'Stephanie Aveldanez',
+    emailAddress: 'stepha@gmail.com',
+    personalPhoneNumber: 619888888,
+    homeAddress: '12345 Fake Street San Diego, CA 92139',
+    emergencyContactName: 'Alex Aveldanez',
+    emergencyContactPhone: 7708917842,
+    signUpDate: '1-7-2020',
+    congregationName: 'Southport',
+    groupNumber: 1
+  },
+  {
+    fullName: 'Michelle Ramirez',
+    emailAddress: 'micheller@gmail.com',
+    personalPhoneNumber: 6191234567,
+    homeAddress: '12345 Street San Diego, CA 92139',
+    emergencyContactName: 'Humberto Ramirez',
+    emergencyContactPhone: 6191234567,
+    signUpDate: '1-7-2020',
+    congregationName: 'Southport',
+    groupNumber: 1
+  }
+]
+

```

Comment:
move this into `src/spas/libs/member-management/domain/src/lib/mocks` and export the class via the `member-management-api.ts` or `index.ts`